test/over_react_test/validation_util_test.dart

```diff
@@ -0,0 +1,121 @@
+// Copyright 2017 Workiva Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import 'dart:html';
+
+import 'package:react/react_dom.dart' as react_dom;
+import 'package:over_react_test/over_react_test.dart';
+import 'package:test/test.dart';
+
+import './utils/test_validation_util_component.dart';
+
+/// Main entry point for `validation_util.dart` testing.
+main() {
+  group('ValidationUtil:', () {
+    DivElement mountNode;
+
+    setUp(() {
+      startRecordingValidationWarnings();
+
+      mountNode = new DivElement();
+      document.body.append(mountNode);
+    });
+
+    tearDown(() {
+      stopRecordingValidationWarnings();
+
+      tearDownAttachedNodes();
+      unmount(mountNode);
+      mountNode.remove();
+
+      mountNode = null;
+    });
+
+    group('startRecordingValidationWarnings()', () {
+      test('begins recording validation warnings, appending them to `_validationWarnings` as expected', () {
+        react_dom.render(TestValidationWarnings()(), mountNode);
```

Comment:
Using a component to test this seems really unnecessary. Why not just call `ValidationUtil.warn` directly in this test?